classes/opponentPhaseBullethell1.lua

```diff
@@ -1,4 +1,4 @@
--- The template opponent phase
+-- Intro to bullet hell
```

Comment:
don't say h*ll, say heck, this is christan minecraft server :/

(jk you don't actually need to do anything)